dropwizard-logging/src/main/java/io/dropwizard/logging/LoggingUtil.java

```diff
@@ -0,0 +1,71 @@
+package io.dropwizard.logging;
+
+import ch.qos.logback.classic.LoggerContext;
+import io.dropwizard.util.Duration;
+import org.slf4j.ILoggerFactory;
+import org.slf4j.LoggerFactory;
+import org.slf4j.bridge.SLF4JBridgeHandler;
+
+import javax.annotation.concurrent.GuardedBy;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+
+public class LoggingUtil {
```

Comment:
I was thinking it should be public because so that people implementing their own logging factories have a way to get the LoggerContext in a thread safe way and have a utility for hijacking jdk logging.